src/DetailsViewPanel.ts

```diff
@@ -148,28 +142,16 @@ export default class DetailsViewPanel {
     }
 
     private generateNavigationLinks(baseUrl: string) {
-        let result = "";
+        let nav = [];
         if (this.prevHistory.length > 0) {
-            const lastIndex = this.prevHistory.length - 1;
-            const link = this.makeRelative(baseUrl, this.prevHistory[lastIndex]);
-            result += `<a 
-                class="js-history-item history-prev" 
-                data-direction="prev"
-                href="${link}" 
-                title="previous page"></a>`;
+            nav.push(`<a class="js-history-item history-prev" data-direction="prev" href="${this.prevHistory[this.prevHistory.length - 1]}" title="previous page"></a>`);
         }
 
         if (this.nextHistory.length > 0) {
-            const lastIndex = this.nextHistory.length - 1;
-            const link = this.makeRelative(baseUrl, this.nextHistory[lastIndex]);
-            result += `<a 
-                class="js-history-item history-next" 
-                data-direction="next"
-                href="${link}"
-                title="next page"></a>`;
+            nav.push(`<a class="js-history-item history-next" data-direction="next" href="${this.nextHistory[this.nextHistory.length - 1]}" title="next page"></a>`);
         }
 
-        return result;
+        return nav.join("\n");
```

Comment:
All changes in this file are just to work with the new results.  Also switched to using an array which gets joined vs string concatenation.  Just personal preference here as I think it's a little easier to ready, but happy to change it back if you prefer it the other way.